config.json

```diff
@@ -71,6 +71,19 @@
         "difficulty": 2,
         "status": "active"
       },
+      {
+        "slug": "tournament",
+        "name": "Tournament",
+        "uuid": "abc1dd61-d677-4e7c-95a6-b9c6f797b2bd",
+        "practices": [],
+        "prerequisites": [],
+        "difficulty": 5,
+        "topics": [
+          "arrays",
+          "sorting",
+          "text_formatting"
+        ]
```

Comment:
We're not using this key in v3.

Reply:
Meh. Already merged. Should I drop this in another PR?